frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/queries.ts

```diff
@@ -0,0 +1,48 @@
+import * as _ from 'lodash';
+
+export enum VMQueries {
+  CPU_USAGE = 'CPU_USAGE',
+  MEMORY_USAGE = 'MEMORY_USAGE',
+  FILESYSTEM_USAGE = 'FILESYSTEM_USAGE',
+  NETWORK_INOUT_USAGE = 'NETWORK_INOUT_USAGE',
+  // NETWORK_IN_USAGE = 'NETWORK_IN_USAGE',
+  // NETWORK_OUT_USAGE = 'NETWORK_OUT_USAGE',
+}
+
+const queries = {
+  [VMQueries.CPU_USAGE]: _.template(
+    // TODO verify; use seconds or milicores?
+    // `kubevirt_vmi_vcpu_seconds{exported_namespace='<%= namespace %>',name='<%= vmName %>'}<%= duration %>`,
+    `pod_name:container_cpu_usage:sum{namespace='<%= namespace %>',pod_name='<%= launcherPodName %>'}<%= duration %>`,
```

Comment:
Please remove `duration` from these queries and add it later in component. When you use query with time selector it will break prometheus URL link

Reply:
done